sdk/communication/Azure.Communication.JobRouter/samples/JobQueueCrud.md

```diff
@@ -0,0 +1,79 @@
+# Azure.Communication.JobRouter Samples - Job Queue CRUD operations (sync)
+
+## Import the namespaces
+
+```C# Snippet:Azure_Communication_JobRouter_Tests_Samples_UsingStatements
+using Azure.Communication.JobRouter;
+```
+
+## Create a client
+
+Create a `RouterClient`.
+
+```C# Snippet:Azure_Communication_JobRouter_Tests_Samples_CreateClient
+var routerClient = new RouterClient(Environment.GetEnvironmentVariable("AZURE_COMMUNICATION_SERVICE_CONNECTION_STRING"));
```

Comment:
We generally use placeholder slugs for connection strings in samples rather than complicate things by introducing environment variables.   Example:

```csharp
var routerClient = new RouterClient("<< CONNECTION STRING >>");
```

Reply:
This has been updated